app/src/main/java/ch/epfl/skysync/navigation/HomeGraph.kt

```diff
@@ -34,76 +29,42 @@ fun NavGraphBuilder.homeGraph(
     composable(Route.CHAT) { ChatScreen(navController) }
     composable(Route.FLIGHT) { FlightScreen(navController) }
     composable(Route.HOME) {
-      val flightsViewModel =
-          FlightsViewModel.createViewModel(
-              flightTable = repository.flightTable,
-              balloonTable = repository.balloonTable,
-              basketTable = repository.basketTable,
-              flightTypeTable = repository.flightTypeTable,
-              vehicleTable = repository.vehicleTable)
+      val flightsViewModel = FlightsViewModel.createViewModel(repository)
+      flightsViewModel.refresh()
```

Comment:
flightsViewModel should be a shared view model

Reply:
For now we don't really need it but maybe at some point we can switch to a shared model. This requires some restructuring of the nav graph though